Zuva DocAI.postman_collection.json

```diff
@@ -25,6 +24,9 @@
 									"pm.test(\"The response contains a file_id.\", function () {",
 									"    pm.expect(jsonData).to.have.property('file_id');",
 									"    pm.environment.set(\"file_id\", jsonData[\"file_id\"]);",
+									"",
+									"    // Chaining for when we run the whole collection",
+									"    postman.setNextRequest(\"Create OCR Request\")",
```

Comment:
Just making a mental note that they are chained by name so if we ever rename any of the requests we'll have to remember to update these as well.